pom.xml

```diff
@@ -295,7 +295,7 @@
           <dependency>
             <groupId>net.trajano.wagon</groupId>
             <artifactId>wagon-git</artifactId>
-            <version>2.0.3</version>
+            <version>2.0.4</version>
           </dependency>
```

Comment:
@allantodd move wagon-git back to 2.0.3.  The release will fail if 2.0.4 for 2 reasons.  1 it was coded wrong and 2 its defunct and won't be fixed.  Only 2.0.3 is valid.

Reply:
done